README.md

```diff
@@ -11,41 +11,24 @@ which can be ran by following the [README.md file](https://github.com/grafana/gr
 
 ### Core
 
-[@grafana/javascript-agent-core](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/core) is the
+[@grafana/agent-core](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/core) is the
 main package that provides the core functionality of the agent. The [README.md](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/core/README.md)
 file provides an overview of the architecture and the API of library.
 
-### Instrumentations
+### Web
 
-1. [@grafana/javascript-agent-instrumentation-console](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/instrumentation-console)
-   is a package for both, browsers and Node.js, which provides an automatic mechanism for collecting console events.
-1. [@grafana/javascript-agent-instrumentation-errors](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/instrumentation-errors)
-   is a browser package which provides an automatic mechanism for collecting unhandled exceptions and errors.
-1. [@grafana/javascript-agent-instrumentation-tracing](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/instrumentation-tracing)
-   is a package for both, browsers and Node.js, which provides an automatic mechanism for tracing. FOR THE TIME BEING
-   THIS PACKAGE IS A PLACEHOLDER.
-1. [@grafana/javascript-agent-instrumentation-web-vitals](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/instrumentation-web-vitals)
-   is a browser package which collects [web vitals](https://web.dev/vitals/) metrics from the app.
-
-### Meta
+[@grafana/agent-web](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/web) provides instrumentations, metas and transports for use in web applications. See The [README.md](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/web/README.md) for more information.
 
-1. [@grafana/javascript-agent-meta-browser](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/meta-browser)
-   is a browser package for collecting details about the browser, operating system and device type.
-1. [@grafana/javascript-agent-meta-page](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/meta-page)
-   is a browser package for collecting details about the current page (i.e. the URL, page title etc.).
+### Web-tracing
 
-### Transports
+[@grafana/agent-web-tracing](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/web-tracing) provides implementation for tracing web applications. See The [README.md](https://github.com/grafana/grafana-javascript-agent/tree/main/packages/web-tracing/README.md) for more information.
```

Comment:
1. I would consider this part of the `Web` section.
2. `web-tracing` vs `tracing-web`. I would advocate for the latter.

Reply:
- [ ] rename to `tracing-web`